app/models/ingest_job.rb

```diff
@@ -246,7 +246,7 @@ def set_study_state_after_ingest
       self.study.set_gene_count
     when 'Cluster'
       self.set_study_default_options
-      self.launch_subsample_jobs
+      #self.launch_subsample_jobs
```

Comment:
Same here - should not be commented out.

Reply:
Addressed by updating with Devon's recent merge, b5db36c.